requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
 pytest-django==4.2.0
 flake8==3.9.1
+factory-boy==3.2.0
```

Comment:
Try to have the requirements files sorted, not important but easier to look at 😉